pepper-apis/src/main/resources/org/broadinstitute/ddp/db/dao/AnswerDao.sql.stg

```diff
@@ -43,6 +50,8 @@ select ans.answer_id,
   left join picklist_option as po on po.picklist_option_id = pa.picklist_option_id
   left join picklist_nested_option as npo on npo.nested_option_id = pa.picklist_option_id
   left join picklist_option as ppo on ppo.picklist_option_id = npo.parent_option_id
+  left join picklist_grouped_option as gpo on gpo.picklist_option_id = pa.picklist_option_id
+  left join picklist_group as plg on plg.picklist_group_id = gpo.picklist_group_id
```

Comment:
We can remove these two left joins since we have the subquery.

Reply:
damn. How did I miss this deletion. thank you